app/1.0/docs/devguide/properties.md

```diff
@@ -732,6 +732,45 @@ nameChanged: function(newFirstName, oldFirstName) {
 Note that Polymer doesn't guarantee that properties are
 initialized in any particular order.
 
+Be very careful when changing the observed value to avoid an infinite loop.
```

Comment:
This addition should be extracted into a separate section, since it is not strictly related to this section ("Always include dependencies as observer arguments"). Could be "Modifying the observed value in an observer" or "Avoid infinite loops."